pack/tsk/tskp.json

```diff
@@ -1,81 +1,542 @@
+        "traits": "Spell.",
+        "type_code": "event",
+        "xp": 0
+    },
+    {
+        "code": "09082",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Steve Hamilton",
+        "name": "Ceremonial Sickle",
+        "pack_code": "tskp",
+        "position": 82,
+        "quantity": 2,
+        "skill_combat": 1,
+        "text": "[action]: <b>Fight.</b> Either use your [willpower] or get +1 [combat] for this attack. When you initiate this ability, choose one - \n- Exhaust Ceremonial Sickle and place 1 doom on it to get +1 skill value and deal +1 damage for this attack.\n- If this attack defeats an enemy, remove 1 doom from Ceremonial Sickle.",
+        "traits": "Item. Charm. Weapon. Melee.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "09083",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Tiziano Baracchi",
+        "name": "Dowsing Rod",
+        "pack_code": "tskp",
+        "position": 83,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "text": "[action]: <b>Investigate.</b> Either use your [willpower] or get +1 [intellect] for this investigation. When you initiate this ability, choose one - \n- Exhaust Dowsing Rod and place 1 doom on it to move to a connection location.\n- If this investigation discovers the last clue at a location, remove 1 doom from Dowsing Rod.",
+        "traits": "Item. Charm.",
+        "type_code": "asset",
+        "xp": 0
+    },
+
+    {
+        "code": "09099",
+        "cost": 3,
+        "customization_options": [
+            {
+                "xp": 1
+            },
+            {
+                "xp": 1
+            },
+            {
+                "xp": 2
+            },
+            {
+                "xp": 2
+            },
+            {
+                "xp": 2
+            },
+            {
+                "xp": 3
+            },
+            {
+                "xp": 4
+            }
+        ],
+        "customization_text": "□ <b>Detachable.</b> Other investigators at your location may use the ability on Pocket Multi Tool.\n□ <b>Pry Bar.</b> You get an additional +1 skill value if this is during a skill test on a treachery.\n□□ <b>Sharpened Knife.</b> You get an additional +1 skill value if this is during an attack.\n□□ <b>Signal Mirror.</b> You get an additional +1 skill value if this is during an evasion attempt.\n□□ <b>Magnifying Lens.</b> You get an additional +1 skill value if this is during an investigation.\n□□□ <b>Lucky Charm.</b> After you fail a skill test, ready Pocket Multi Tool.\n□□□ <b>Spring-Loaded.</b> Pocket Multi Tool's ability is now a [reaction] ability with the trigger: \"When you would fail a skill test you are performing, exhaust Pocket Multi Tool...\"",
```

Comment:
```suggestion
        "customization_text": "□ <b>Detachable.</b> Other investigators at your location may use the ability on Pocket Multi Tool.\n□ <b>Pry Bar.</b> You get an additional +1 skill value if this is during a skill test on a treachery.\n□□ <b>Sharpened Knife.</b> You get an additional +1 skill value if this is during an attack.\n□□ <b>Signal Mirror.</b> You get an additional +1 skill value if this is during an evasion attempt.\n□□ <b>Magnifying Lens.</b> You get an additional +1 skill value if this is during an investigation.\n□□□ <b>Lucky Charm.</b> After you fail a skill test, ready Pocket Multi Tool.\n□□□ <b>Spring-Loaded.</b> Pocket Multi Tool's ability is now a [reaction] ability with the trigger: \"When you would fail a skill test you are performing, exhaust Pocket Multi Tool...\"",
```

Reply:
Done